hummingbot/connector/exchange/ndax/ndax_exchange.py

```diff
@@ -763,25 +763,25 @@ async def _update_order_status(self):
             else:
                 self.logger().error(f"Error fetching order status. Response: {resp}")
 
-        min_ts: int = min([int(order_status["ReceiveTime"] // 1e3)
+        min_ts: int = min([int(order_status["ReceiveTime"])
                            for order_status in parsed_status_responses])
 
         trade_history_tasks = []
         trading_pair_ids: Dict[str, int] = await self._order_book_tracker.data_source.get_instrument_ids()
 
         for trading_pair in self._trading_pairs:
-            query_params = {
+            body_params = {
                 "OMSId": 1,
                 "AccountId": await self.initialized_account_id(),
                 "UserId": self._auth.uid,
                 "InstrumentId": trading_pair_ids[trading_pair],
                 "StartTimestamp": min_ts,
-                "EndTimestamp": int(time.time()),
+                "EndTimestamp": int(time.time() * 1e3),
```

Comment:
Since we want all the trades history up to now, and there could be differences between the client time and the exchange time, I think it would be better to not include the EndTimestamp parameter (ie: let the exchange return all the latest information)

Reply:
Yes I agree that `EndTimestamp` is essentially redundant and might not be an accurate representation of the orders we want to capture due to the difference in local and server time. I will make the necessary changes.